keyboards/handwired/axon/keymaps/default/keymap.c

```diff
@@ -0,0 +1,26 @@
+#include "axon.h"
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

Reply:
Resolved in commit 7a0f99f: https://github.com/qmk/qmk_firmware/pull/12879/commits/7a0f99fe977601d849683a9f77a6aefd1e8f33bb